templates/graph.html

```diff
@@ -0,0 +1,21 @@
+{% extends "page.html" %}
+{% block head %}
+  <title>Home page | F# Snippets</title>
+{% endblock %}
+
+{% block customPageScripts %}
+<script type="text/javascript" src="https://www.google.com/jsapi"></script>
```

Comment:
For server side usage and IDE integration I left loading the appropriate visualization packages to the user. According to the screenshots below this wasn't a showstopper for you.